.circleci/config.yml

```
@@ -99,8 +99,8 @@ jobs:
             ./.circleci/submodule.sh
       - restore_cache:
           keys:
-          - v1-build-latest-yarn-{{ checksum "package.json" }}
-          - v1-build-latest-yarn-
+          - v1-test-{{ checksum "package.json" }}
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 12 but found 10

(indentation)